operator/src/test/java/org/keycloak/operator/testsuite/integration/WatchedSecretsTest.java

```diff
@@ -78,6 +76,47 @@ public void testSecretsAreWatched() {
         });
     }
 
+    @Test
+    public void testClonedSecretUnmodified() {
```

Comment:
I wonder if we need this test after all. We completely removed the logic that touches watched (or cloned) secrets.

Technically, it's also not testing the cloned secret (as the name suggests). The informer is watching the normal (non-cloned) tls secret.

Reply:
Right it needs to be watching both, with the creation only on example-tls-secret. I thought I had made that change at some point (I grabbed this code from the older draft pr, but didn't heavily review it first).

I'm fine with making that change, or removing if you are good with that.